spec_linux.go

```diff
@@ -123,9 +123,13 @@ func createLibcontainerConfig(spec *LinuxSpec) (*configs.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	var rootfsPath string = spec.Root.Path
```

Comment:
Sorry for the nit but can you please change this line to:

`rootfsPath := spec.Root.Path`

?